src/web/app/components/session-edit-form/session-edit-form.component.ts

```diff
@@ -88,6 +88,9 @@ export class SessionEditFormComponent implements OnInit {
   @Output()
   editExistingSessionEvent: EventEmitter<void> = new EventEmitter<void>();
 
+  @Output()
+  cancelExistingSessionEvent: EventEmitter<void> = new EventEmitter<void>();
```

Comment:
Shouldn't it be `cancelEditingSessionEvent`?

Reply:
That makes sense as well. I'll change it to `cancelEditingSessionEvent`